source/javascripts/terminology_tooltip.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+[...document.getElementsByClassName('terminology')].forEach($terminology => {
+    const horizontalMargin = 20;
+
+    const topMargin = document
+        .getElementsByClassName('site-header')[0]
+        .clientHeight;
+
+    const terminology = $terminology.innerText;
+
+    const debug = message => {
```

Comment:
This do not need to be here.

Reply:
I understand you want to me to get rid of `console.debug()`, not just `debug()` helper? ;)

Reply:
Yeah, both of them 👍

Reply:
https://github.com/home-assistant/home-assistant.io/pull/31355/commits/aa23edcb8f62957c2bfa1c6b416415dedd81d254